js/context/UserContext/UserProvider.js

```diff
@@ -59,7 +77,12 @@ class UserProvider extends Component {
           addUserToken: this.addUserToken,
           removeUserToken: this.removeUserToken,
           updateUser: this.updateUser,
-          getUser: this.getUser
+          getUser: this.getUser,
+          addActivity: this.addActivity,
+          removeActivity: this.removeActivity,
+          addJournal: this.addJournal,
+          removeJournal: this.removeJournal,
+          firstTime: this.firstTime
```

Comment:
firstTime what? Please name your props better.

Reply:
changed to 2 different methods named toggleShowOnBoarding and toggleShowPrivacyPolicy